### DISCUSSION TITLE
Can we have only year in DateTimePicker field?

### DISCUSSION THREAD

**Discussion by amansaini419** [filamentphp/filament#12003] [Ideas]
I tried doing this by following:

![image](https://github.com/filamentphp/filament/assets/6827644/d5d32c8e-137a-4ed6-99ff-c6aee43867d2)

![image](https://github.com/filamentphp/filament/assets/6827644/1b3e076b-1c4e-4ef4-9979-5b5d308e4ac4)

![image](https://github.com/filamentphp/filament/assets/6827644/1199bcd5-e6df-4729-9ce7-da8a3c4ce131)

![image](https://github.com/filamentphp/filament/assets/6827644/a29e0821-172a-4bfa-9d07-e6f6654e1a12)

I only want to see year dropdown, no month or date. Can this be achieved with the current features?

**Comment by leandrocfe** [comment]
https://github.com/filamentphp/filament/discussions/11971
